Example/Showcase/Classes/PresentationLayer/Web3Inbox/Web3InboxViewController.swift

```diff
@@ -18,9 +18,19 @@ final class Web3InboxViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        Web3Inbox.configure(account: account)
+        Web3Inbox.configure(account: importAccount.account, onSign: onSing)
         view = Web3Inbox.instance.getWebView()
 
         navigationItem.title = "Web3Inbox SDK"
     }
 }
+
+private extension Web3InboxViewController {
+
+    func onSing(_ message: String) -> SigningResult {
```

Comment:
why we eventually need this?

Reply:
because we need to sign cacao from Chat SDK register method